### DISCUSSION TITLE
Source "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol"  file not found.

### DISCUSSION THREAD

**Discussion by MohiniVasisth** [Cyfrin/foundry-full-course-cu#1227] [Q&A]
Hi sir 
I follows the same steps as you shown in the video .But after installing and remapping creation in foundry.toml . I got this issue . Can you please provide suggestions for fixing this issue.
![Screenshot (89)](https://github.com/Cyfrin/foundry-full-course-f23/assets/98387326/2544a284-8b31-4fc5-9448-c82c4c5e624a)

**Comment by xyz899** [comment]
Hi,
use `forge remappings > remappings.txt` it should remap automatically based on the packages you have, for example it would be written down like this in .txt : 
`openzeppelin/=lib/openzeppelin-contracts/contracts/...`
